utils_test.go

```diff
@@ -15,6 +15,7 @@
 package nutsdb
 
 import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Imports are organized in groups, with blank lines between them. 
  detail see : https://github.com/golang/go/wiki/CodeReviewComments#imports